libconfig/GlobalConfigure.h

```diff
@@ -31,7 +31,8 @@ enum VERSION : uint32_t
 {
     RC1_VERSION = 1,
     RC2_VERSION = 2,
-    RC3_VERSION = 3
+    RC3_VERSION = 3,
+    V2_0_0 = 0x02000000,

```

Comment:
why do you define v2_0_0 as 0x200000?
And why it is v2_0_0 but not v2_1_0?

Reply:
why do you define v2_0_0 as 0x200000?
2.0.0 --->0x02(2) 0x00(0) 0x00（0）0x00(reserved) , we can Compare version through this。

And why it is v2_0_0 but not v2_1_0?
in code,we have two Comparison rules,>2.0.0 or <=2.0.0
of course,we can use >=2.1.0 or <2.1.0 instead.